src/index.js

```diff
@@ -35,17 +31,19 @@ export default class SplunkEvents {
     });
   }
 
-  logEvent(event) {
+  logEvent(host, level, type, workflowType, workflowInstance, event) {
```

Comment:
@Caldas what if we let the user configure a default value for these options?

you can change this line to:
`logEvent({ host, level, type, workflowType, workflowInstance, event }) {`

and extend the configuration values with these specifics params